integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -25,6 +25,20 @@
     "weaviate.auth.AuthApiKey": weaviate.auth.AuthApiKey,
 }
 
+# This is the default collection properties for Weaviate.
+# It's a list of properties that will be created on the collection.
+# These are extremely similar to the Document dataclass, but with a few differences:
+# - `id` is renamed to `_original_id` as the `id` field is reserved by Weaviate.
+# - `blob` is split into `blob_data` and `blob_mime_type` as it's more efficient to store them separately.
+DOCUMENT_COLLECTION_PROPERTIES = [
+    {"name": "_original_id", "dataType": ["text"]},
+    {"name": "content", "dataType": ["text"]},
+    {"name": "dataframe", "dataType": ["text"]},
+    {"name": "blob_data", "dataType": ["blob"]},
+    {"name": "blob_mime_type", "dataType": ["text"]},
+    {"name": "score", "dataType": ["number"]},
+]
+
```

Comment:
Is `meta` field missing? Is it intentional?

Reply:
Yep, it's intentional. Ideally I would specify it like this:
```
{"name": "meta", "dataType": ["object"]}
```
Problem is that that makes it mandatory for `meta` to contain some field. Trying to save a `Document` with `meta == {}` would fail.

If I omit it from the schema and let Weaviate automatically pick the type, that makes it work. I need to verify what happens with `auto-schema` off though.